### ISSUE TITLE
Error complaining that `date-fns` is not found, but I'm using `moment`

### ISSUE THREAD

**Issue by adam-nielsen** [dmtrKovalenko/material-ui-pickers#252]
## Environment

| Tech                | Version |
|---------------------|---------|
| material-ui-pickers | 1.0.0-rc.1 |
| material-ui         | 1.0.0-beta32 |
| React               | 16.2.0  |
| Browser             | N/A     |
| Platform            | Linux |

## Steps to reproduce
1. `import { MuiPickersUtilsProvider } from 'material-ui-pickers';`
2. Get error: 
```
./node_modules/material-ui-pickers/utils/date-fns-utils.js
Module not found: Can't resolve 'date-fns/addDays' in 'node_modules/material-ui-pickers/utils'
```
## Expected behavior
No error

## Actual behavior
Complains that `date-fns` cannot be loaded.  It's not in my project as I'm using `moment`.  The `README` says that `date-fns` is optional and that `moment` can be used instead, but this doesn't appear to work.

**Comment by dmtrKovalenko** [comment]
Yep. We will fix that today

**Comment by dmtrKovalenko** [comment]
Fixed in 1.0.0-rc.2
